pygfx/helpers/_logo.py

```diff
@@ -0,0 +1,46 @@
+def make_logo_scene_camera(size=100, edge_width=3, padding=(5, 5)):
+    from .. import (
+        Group,
+        Geometry,
+        Points,
+        PointsMarkerMaterial,
+        ScreenCoordsCamera,
+    )
+
+    geometry = Geometry(positions=[[size / 2, size / 2, 0]])
+    logo_inner = Points(
+        geometry,
+        PointsMarkerMaterial(
+            size=size,
+            color="#387EB9",
+            marker="pygfx_inner",
+            edge_color="#000",
+            edge_width=edge_width,
+        ),
+    )
+    line_outer = Points(
+        geometry,
+        PointsMarkerMaterial(
+            size=size,
+            color="#FFE64B",
+            marker="pygfx_outer",
+            edge_color="#000",
+            edge_width=edge_width,
+        ),
+    )
+    logo = Group()
+    logo.add(logo_inner, line_outer)
```

Comment:
I'm honestly not so sure why the logo is getting rendered in the "correct" order. Is there some way we can "guarantee" it?